### PR TITLE
Chameleon recipe added

### PR DESCRIPTION
**Pull request by ReimarBauer**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/Chameleon/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    - chameleon
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://chameleon.readthedocs.io
+  summary: Fast HTML/XML Template Compiler.
+  license: BSD
```
Comment (xhochy) [inline recipes/Chameleon/meta.yaml:35]: Please specify which BSD license it is.

Comment (ReimarBauer) [reply recipes/Chameleon/meta.yaml:35]: on pypi it notes This software is made available as-is under a BSD-like license [1] (see included copyright notice).

[1] This software is licensed under the "Repoze" license.

I

File: recipes/Chameleon/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    - chameleon
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://chameleon.readthedocs.io
+  summary: Fast HTML/XML Template Compiler.
+  license: Repoze
```
Comment (xhochy) [inline recipes/Chameleon/meta.yaml:35]:
```suggestion
  license: BSD-4-clause
```

**Comment by xhochy** [comment]
It actually looks extactly like BSD-4-clause which is then a valid SPDX classifier.

**Comment by wolfv** [comment]
@ReimarBauer you can find a list of valid license identifiers here https://conda-forge.org/docs/maintainer/adding_pkgs.html#spdx-identifiers-and-expressions

**Comment by ReimarBauer** [comment]
> 

I know, but there is no BSD-4-clause on that list. I try it asap.

**Comment by ReimarBauer** [comment]
The spdx is given. The list in conda-forge description are examples of the whole list. May be add  as subtitle "A small selection from the list" there.
 
https://spdx.dev/spdx-specification-21-web-version/#h.cy4losc41vio

**Comment by ReimarBauer** [comment]
@conda-forge-admin please have a look, if it is ready to get merged.

**Comment by wolfv** [comment]
congrats! Feel free to send a PR to amend the docs :)

**Pull request merged**